presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcRecordCursor.java

```diff
@@ -217,7 +218,7 @@ public void close()
             // do nothing
         }
         catch (SQLException e) {
-            throw new RuntimeException(e);
+            // ignore exception from close
```

Comment:
Why is this needed?
Normally you do not want to suppress exceptions even in close. Maybe `MysqlClient`'s iml of `abortReadConnection` should swallow exc instead?

Reply:
I agree normally you want to do something with this exception, but in this case MySQL throws a bogus (untyped) exception from close.  Also, in general, there is nothing interesting here for JDBC clients.

Reply:
We discussed this and agreed there was no reason to spam the logs with stack traces from close exceptions. This is for a read, and at this point the data has already been read (or not due to an error), so can anything useful come from the exception?

Reply:
In case of connection used once and only for a select query -- probably nothing useful. Unless it mean that a) a connection was not returned to a pool for some reason (when jdbc connector has pooling) or b) we're not terminating session correctly some times and leaking sessions on the db server (you know, db servers are sometimes stateful).

So -- i still see value in not swallowing exception here, but I am fine if this stays as-is.